ansible_rulebook/cli.py

```diff
@@ -140,14 +145,12 @@ def get_parser() -> argparse.ArgumentParser:
 
 
 def show_version() -> NoReturn:
-    java_home = f"{os.environ.get('JAVA_HOME', 'JAVA_HOME not present')}"
-
     result = [
         f"{ansible_rulebook.__version__}",
         f"  Executable location = {sys.argv[0]}",
         f"  Drools_jpy version = {importlib.metadata.version('drools_jpy')}",
-        f"  Java home = {java_home}",
-        f"  Java version = {get_java_version()}",
+        f"  Java home = {util.get_java_home()}",
+        f"  Java version = {util.get_java_version()}",
```

Comment:
Minor: I think calling functions inside f-strings hurts readability.

Reply:
I buy it. Changed